.env.example

```
@@ -46,6 +46,9 @@ SENTRY_PROJECT=gitpulse
 # SENTRY_AUTH_TOKEN=your-auth-token-from-sentry (required for releases, set in GitHub secrets)
 # Convex Deploy Keys (set in Vercel; preview uses preview:..., production uses prod:...)
 CONVEX_DEPLOY_KEY=your_convex_deploy_key
+# PostHog Analytics
+NEXT_PUBLIC_POSTHOG_KEY=your_posthog_project_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_KEY key should go before the NEXT_PUBLIC_SENTRY_DSN key

(UnorderedKey)